addons/website_event_meet/views/event_type_views.xml

```diff
@@ -5,13 +5,15 @@
         <field name="model">event.type</field>
         <field name="inherit_id" ref="website_event.event_type_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='community-menu']" position='after'>
-                <div class="row mt16" attrs="{'invisible': [('website_menu', '=', False)]}">
-                    <label class="col" for="meeting_room_allow_creation"/> <field name="meeting_room_allow_creation"/>
-                </div>
+            <xpath expr="//field[@name='community_menu']" position='after'>
+                <span
+                     attrs="{'invisible': [('community_menu', '=', False)]}">
```

Comment:
HumanLinter: could be one-lined (no need to set attrs at next line)

Reply:
Done